### DISCUSSION TITLE
Show active alerts for integrations

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/frontend#7435] [Configuration]
We should include the alerts from https://alerts.home-assistant.io/

If there are integrations with alerts, we should show them on the integrations tab. 

The data is available at https://alerts.home-assistant.io/alerts.json

**Comment by spacegaier** [comment]
In addition to the info on the integrations page, wouldn't a notification also make sense to notify the user of the alert, without having to manually check the integrations page?
